### PR TITLE
Data integrity tests

### PR DESCRIPTION
**Pull request by pavelzag**
Purpose or Intent
=================
This test checks correct population of the Properties Fields in Containers/Image Registries/Container Images and Container Nodes details menu

image_registry.py file was edited to allow get_detail on this object

{{pytest: cfme/tests/containers/test_containers_data_integrity_properties.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 47d97f32ec6101c2b96a138d531e5a8c66d3154b:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by pavelzag** [comment]
The tests will fail because they use the history module from 
https://github.com/ManageIQ/integration_tests/pull/3338
When it's merged it shall work

File: cfme/tests/containers/test_containers_data_integrity_properties.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    """ Properties fields test in Containers
+        This test checks correct population of the Properties Fields in Containers' details menu
+        Steps:
+            * Goes to Containers -- > Containers menu
+            * Go through each Container in the menu and check validity of Properties fields
+        """
+    sel.force_navigate('containers_containers')
+    container_name = [r.name.text for r in list_tbl.rows()]
+    container_pod_name = [r.pod_name.text for r in list_tbl.rows()]
+    dictionary = dict(zip(container_name, container_pod_name))
```
Comment (psav) [inline cfme/tests/containers/test_containers_data_integrity_properties.py]: This code is also cropping up in other container PRs, I strongly recommend you guys get together and make a ```get_all_pods``` method for your provider object, which will return for you a list of Pod objects. Or is this something different?

Comment (pavelzag) [inline cfme/tests/containers/test_containers_data_integrity_properties.py]: Can you give me an example of another PR where it happens? I'll check what I can do about it

Comment (mfalesni) [inline cfme/tests/containers/test_containers_data_integrity_properties.py]: Yeah, I have a strong illusion that all the Container PRs add the same code when I am looking at them so I am lost in that. Pete made a good point. What repeats, needs DRY :)

Comment (pavelzag) [inline cfme/tests/containers/test_containers_data_integrity_properties.py]: I understand what you want me to do, but I am not sure where can I stick that part of the code that should scan through content of the table and give me a list of all the containers/pods/images in the table. Should it be in Provider object? Is there something similar in the code? I don't know the infra code enough to decide where to stick it.

Comment (psav) [inline cfme/tests/containers/test_containers_data_integrity_properties.py]: Yup that's exactly what I think would be a good idea. Container Provider object has a ```get_all_pods``` method that returns a list of Pod objects.

Comment (pavelzag) [inline cfme/tests/containers/test_containers_data_integrity_properties.py]: I'll try to concoct something, thanks

**Comment by pavelzag** [comment]
Replaced by https://github.com/ManageIQ/integration_tests/pull/3367

**Pull request closed without merging**